server/data/data-jmap-postgres/src/test/java/org/apache/james/jmap/postgres/upload/PostgresUploadServiceTest.java

```diff
@@ -73,4 +75,11 @@ public UploadUsageRepository uploadUsageRepository() {
     public UploadService testee() {
         return testee;
     }
+
+    @Override
+    @Test
+    @Disabled("Flaky test. TODO stabilize it.")
+    public void uploadShouldUpdateCurrentStoredUsageUponCleaningUploadSpace() {
```

Comment:
Ok but linagora/james-project issue please.
This shall be addressed.

Reply:
https://github.com/linagora/james-project/issues/5058